addons/mail_bot/static/src/messaging/component/messaging_menu/messaging_menu.js

```diff
@@ -17,7 +17,7 @@ patch(components.MessagingMenu, 'mail_bot.messaging.component.MessagingMenu', {
      * @private
      */
     _onOdoobotRequestClicked() {
-        const device = this.env.messaging.device;
+        const device = this.messagingMenu.messaging.device;
```

Comment:
I prefer `this.env.messaging`

Reply:
Out of scope? I don't need it and other mocks just return dummy promises too.

Reply:
I prefer to change it but it just highlights bad code anyway.

All clicks on the messaging menu notif/previews should close the menu, there should not be an override needed for this in `mail_bot`, and also the code should be in entity and not component...

I guess this should be part of clean task-2258605.